### PR TITLE
Add anonymised description field to investment projects

### PR DESCRIPTION
**Pull request by reupen**
Adds the anonymised description field to investment projects. Also enables the working acceptance tests for investment projects after the fixtures in Leeloo were updated.

# Screenshots

screenshot-2017-10-25 department for international trade 1

&nbsp;

screenshot-2017-10-25 department for international trade

### CODE REVIEW

File: test/acceptance/features/investment-projects/create.feature [hunk 1]
```diff
@@ -11,7 +11,7 @@ Feature: Create a new Investment project
 
     When I navigate to Investments page of any company
```
Comment (feedmypixel) [inline test/acceptance/features/investment-projects/create.feature]: I'm not overly happy about this relying on a company being in fixtures, incase:
- other tests change it and expected values (meaning we then have to think about tests order)
- fixtures change

Can you create a company to tests against via [company step definitions](https://github.com/uktrade/data-hub-frontend/blob/develop/test/acceptance/features/companies/step_definitions/company.js#L14-L57) examples can be seen in [company create features](https://github.com/uktrade/data-hub-frontend/blob/develop/test/acceptance/features/companies/create.feature)

File: test/acceptance/features/investment-projects/create.feature [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@investment-projects-create @ignore
+@investment-projects-create
```
Comment (feedmypixel) [inline test/acceptance/features/investment-projects/create.feature]: can you add a `@todo` flag on this. This is signify ones we have to revisit post launch

File: src/apps/investment-projects/views/_details-form.njk [hunk 3]
```diff
@@ -42,6 +42,16 @@
...
   }) }}
 
+  {{ TextField({
+    name: 'anonymous_description',
+    label: 'Anonymised description of the project',
+    hint: 'Do not include company names, financial details or address details',
+    type: 'textarea',
+    optional: true,
+    error: form.errors.messages.anonymous_description,
+    value: form.state['anonymous_description']
```
[the remark this answers is not in the document]
Comment (reupen) [reply src/apps/investment-projects/views/_details-form.njk]: Yep – the entire file is inconsistent though.

File: test/acceptance/features/investment-projects/create.feature (hunk 1 above)
Comment (reupen) [reply test/acceptance/features/investment-projects/create.feature]: Well the point of the fixtures is that you can rely on them being there. So the other tests shouldn't be modifying them. If you create companies in these tests, they're all going to fail if something is wrong with creating companies, and it's also going very inefficient.

File: src/apps/investment-projects/views/_details-form.njk (hunk 3 above)
[the remark this answers is not in the document]
Comment (feedmypixel) [reply src/apps/investment-projects/views/_details-form.njk]: > Yep – the entire file is inconsistent though.

something for future refactor

File: test/acceptance/features/investment-projects/create.feature (hunk 1 above)
Comment (feedmypixel) [reply test/acceptance/features/investment-projects/create.feature]: yeah as discussed in slack I think there is a middle ground and this is WIP at present so - fine for the mo, but to be revisited

**Pull request merged**